route/build/cpl/RtmMod.F90

```diff
@@ -429,19 +448,29 @@ SUBROUTINE route_run(rstwr)
       if(ierr/=0)then; call shr_sys_abort(trim(subname)//trim(cmessage)); endif
     enddo
 
+    call t_stopf('mizuRoute_subcycling')
+
+    !-----------------------------------
     ! getting ready for exporting
+    !-----------------------------------
+    call t_startf('mizuRoute_prep_export')
+
+    rtmCTL%discharge = 0._r8
+    rtmCTL%volr      = 0._r8
```

Comment:
It would really be great to see volr get set to real values, rather than just initialized to zero.

Reply:
So here just initialization. These are supposed to be updated in line 540-542 (yes, still I need to set real volume there).  Question is do I need to initialize zero for these at each time step?

Reply:
I wouldn't set to zero each time step, there's no reason to do that. Really you just need the correct value by the time it's sent to the mediator. It's not bad for it to be the previous value before that. It's good to have it set to zero at initialization to make sure it's set everywhere, in case later on not all cells end up being set or something. So explicitly setting it to zero each time step is a waste of CPU time. You might zero something each time step if it's a flux that's going to accumulate over different processes, but outside of that and for most state variables you needn't reinitialize to zero.